railroad/cargo_train.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './train'
+
+class CargoTrain < Train
+  def initialize(number, wagons = [])
+    super(number, wagons)
```

Comment:
```suggestion
    super
```
"супер" без аргументов передает все пришедшие аргументы